recipe/meta.yaml

```diff
@@ -504,6 +504,8 @@ outputs:
     test:
       requires:
         - pip
+        # TODO: clean up after https://github.com/conda-forge/tqdm-feedstock/issues/84
+        - tqdm !=4.53.0
```

Comment:
workaround for https://github.com/conda-forge/tqdm-feedstock/issues/84

Reply:
https://github.com/conda-forge/tqdm-feedstock/pull/85 merged so can revert this constraint

Reply:
Thanks!